packages/cli/src/oclif/hooks/getAppRegistrationFieldChoices.js

```diff
@@ -0,0 +1,17 @@
+const { callAPI } = require('../../utils/api');
+
+module.exports = async function (options) {
+  // We only need to run this for the register command
+  if (!options || !options.id || options.id !== 'register') {
+    return null;
```

Comment:
Good provision to return early for other commands/use-cases except for `register`.